src/utils/h5p.utils.ts

```diff
@@ -1,6 +1,8 @@
-import type { H5PLibrary } from 'h5p-types';
+import type { H5PLibrary, H5PObject } from 'h5p-types';
 import libraryConfig from '../../library.json';
 
+export const H5P: H5PObject = (window as any).H5P ?? {};
```

Comment:
This is already exported from `h5p-utils`.

Reply:
Indeed, thanks. I had not checked all the imports.

Reply:
I have considered `declare`ing `H5P`, `H5PEditor` and `H5PIntegration`, but I'd like not to as some of them are always _not_ defined in some cases.

Reply:
If I am not mistaken, `H5P` should always be defined. Same for `H5PIntegration`, but with different properties depending on whether content is viewed or edited. `H5PEditor` should be set when content is edited.